charts/network-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "hny-network-agent.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
```

Comment:
We need to add these labels to all `metadata.labels` (only the labels at the root of the objects): https://github.com/open-telemetry/opentelemetry-helm-charts/blob/8402aeb044cf47f8f47590a3f094fdb79395a745/charts/opentelemetry-collector/templates/_helpers.tpl#L52-L62

Reply:
Added helpers to manage common and selector labels in https://github.com/honeycombio/helm-charts/pull/292/commits/c322ed778a57a87443e2a0d2c3ffbb962602a895